### PR TITLE
Add ParsedJson.items_from_parsed_query 

### PR DESCRIPTION
**Pull request by tkilias**
An proof-of-concept for #27 which is around 5-10% faster than ParsedJson.items if the query is reused for many json documents. Please have a look. If you think, we could integrate this, I going to add the missing documentation to ParsedJson.items_from_parsed_query.

### CODE REVIEW

File: simdjson/csimdjson.pyx [hunk 1]
```diff
@@ -382,6 +382,13 @@ cdef class ParsedJson:
             raise JSONDecodeError('Error iterating over document', '', 0)
 
         return self._items(iter, parsed_query)
+    
+    def items_from_parsed_query(self, parsed_query):
```
Comment (TkTech) [inline simdjson/csimdjson.pyx]: Rather than a separate method, `items` should accept both a string or a 'parsed query'.

Comment (tkilias) [reply simdjson/csimdjson.pyx]: Ok, sounds good for me. Do we want to use the same parameter for both variants and call the necessary path depending on the type?

I am going to change it after Christmas. I wish you happy holidays.

Comment (TkTech) [reply simdjson/csimdjson.pyx]: Yes exactly :) Lets keep the API simple.
Hope you had a good holiday.

Comment (tkilias) [reply simdjson/csimdjson.pyx]: Yes,thanks, hope, you too. I added the type check to ParsedJson.items. For the time being, I used isinstance. I read that PyString_Check and PyList_Check are faster. However, these functions we would need to call in cdef function and from the speed test it doesn't look like that the isinstance has a major impact on the performance.

**Comment by TkTech** [comment]
The usage of items() is being replaced by simdjson's native support for JSON paths in #33.

**Pull request closed without merging**